Assets/LeapMotion/Scripts/Utils/HandUtils.cs

```diff
@@ -71,7 +71,6 @@ public class HandEvent : UnityEvent<Hand> { }
     public static Hand Left {
       get {
         if (Provider == null) return null;
-        if (Provider.CurrentFrame == null) return null;
```

Comment:
John assured me that Provider never has a null CurrentFrame, so I'm committing to that :P is it not true?